beacon-chain/types/block.go

```diff
@@ -48,7 +49,7 @@ func NewBlock(data *pb.BeaconBlock) *Block {
 
 // NewGenesisBlock returns the canonical, genesis block for the beacon chain protocol.
 func NewGenesisBlock() (*Block, error) {
```

Comment:
I don't think this should return an error when the genesis time is a constant.

Can you 

1) ignore the error and leave a comment that says the genesis time is static and the error can be safely ignored
1) add a test that ensures the GenesisTime constant passes if _, err := ptypes.TimestampProto(GenesisTime); err != nil { t.Errorf(...) }

Reply:
Agreed - it should be something we can call everywhere without fear of error